src/data_snack/serializers/dataclass.py

```diff
@@ -14,14 +14,14 @@ def _serialize(entity: Entity) -> bytes:
 @dataclass
 class DataclassSerializer(Serializer):
     def __post_init__(self):
-        self.entity_fields = list(self.entity_type.__dataclass_fields__.keys())
+        self.entity_fields = list(self.entity_type.__dataclass_fields__.keys())  # noqa
```

Comment:
How this line works? `entity_type` is of type `Entity`, but I can't reporduce example which allows to use `__dataclass_fields__` methods. Is everything correct here in terms of typings?